frontend/src/components/agency/ds-settings-card/DsSettingsCard.jsx

```diff
@@ -133,6 +143,7 @@ function DsSettingsCard({ type, endpointDetails, message }) {
 
   useEffect(() => {
     const menuItems = [];
+    console.log(listOfConnectors, "sdgsdfsdfsd");
```

Comment:
@vishnuszipstack Please remove the console.log

Reply:
done @tahierhussain